.github/workflows/community-comment.yml

```
@@ -0,0 +1,21 @@
+name: Add comment to community issues
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'community'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add comment
+        uses: peter-evans/create-or-update-comment@a35cf36e5301d70b76f316e867e7788a55a31dae
```

```suggestion
        uses: peter-evans/create-or-update-comment@67dcc547d311b736a8e6c5c236542148a47adc3d
```

Any reason to prefer v1 (`a35cf3`) instead of v2 (`67dcc5`)? I reviewed the action code for v2 but I can go back and look at v1 instead if you prefer that

---

Nope. Just the code I copied 😁